### ISSUE TITLE
Error Installing SIFT

### ISSUE THREAD

**Issue by amcap623** [teamdfir/sift#490]
I am trying to install SIFT and am getting this error.  Please help.  

>> Running: sift-config-tools
Update returned exit code not zero
Error: Update returned exit code not zero
    at ChildProcess.<anonymous> (/snapshot/sift-cli/sift-cli.js:529:23)
    at ChildProcess.emit (events.js:196:13)
    at maybeClose (internal/child_process.js:1011:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:268:5)

----- PLEASE READ ----------------------

A lot of failures are caused by the apt system being locked
or unhealthy. 

Before opening an issue in GitHub, please check to see if 
your apt system is healthy.

Try running 'apt-get update' then remove any packages that 
aren't used by running 'apt-get autoremove'

**Comment by ekristen** [comment]
Please provide the full saltstack log. It's likely that just a single thing is failing.

**Comment by n00btotal** [comment]
I have the same problem.

Looks like a problem with gsettings can't find schemas?

But I do get several more errors (result: false)

`  ? cmd_|-sift-config-theme-gnome-launcher-position_|-gsettings set org.gnome.shell.extensions.dash-to-dock
    dock-position BOTTOM_|-run
  : __id__: sift-config-theme-gnome-launcher-position
    __run_num__: 519
    __sls__: sift.config.user.theme_bionic
    changes:
      pid: 15378
      retcode: 1
      stderr: No such schema ?org.gnome.shell.extensions.dash-to-dock?
      stdout: ''
    comment: Command "gsettings set org.gnome.shell.extensions.dash-to-dock dock-position
      BOTTOM" run
    duration: 474.757
    name: gsettings set org.gnome.shell.extensions.dash-to-dock dock-position BOTTOM
    result: false`


[saltstack.log](https://github.com/teamdfir/sift/files/5187041/saltstack.log)

**Comment by amcap623** [comment]
The latest releases site had version 1.8.5 ..  Showed these 2 commands..  so I ran them and then ran the sudo sift install again which worked. 

RESULT::  >> COMPLETED SUCCESSFULLY -- Success: 571, Failure: 0

curl -Lo /usr/local/bin/sift https://github.com/sans-dfir/sift-cli/releases/download/v1.8.5/sift-cli-linux
chmod +x /usr/local/bin/sift

**Comment by ekristen** [comment]
@n00btotal this seems to be a bug/problem with gsettings and gnome possibly. Is this a vanilla Ubuntu 18.04 Desktop install or has it been modified in any way? 

@amcap623 glad to hear it is working for you now.

**Comment by n00btotal** [comment]
@ekristen  it is a vanilla ubuntu desktop (from minimal netboot install image). I made no modifications or updates before I tried the sift installation

**Comment by gebhard73** [comment]
Got a similar error with saltstack. Please see pull-request https://github.com/teamdfir/sift-cli/pull/50, this may help.

**Comment by n00btotal** [comment]
From a fresh minimal ubuntu install (I added no extra packages during install, i.e no desktop environment) I successfully installed sift with 
`sift install --mode=server --user=<the user I added during install>`

When I tried to update with --mode=desktop the update fails.

I installed the default desktop environment with
`sudo tasksel install ubuntu-desktop`

After a reboot and login, I'm greeted with the SIFT workstation background and what it seems all the desktop icons.

Then, after running 
`sudo sift update`
I get 
`>> COMPLETED SUCCESSFULLY -- Success: 571, Failure: 0`

May I suggest that you add a note about that the default desktop environment is required. If there is one, I missed it. (I did notice that "lubuntu" is not supported, which led me to believe that I must have a desktop environment in order to successfully run the installation process for sift).
